melos.yaml

```diff
@@ -22,3 +22,6 @@ scripts:
   format-check: dart format --output=none --set-exit-if-changed --line-length 120 .
   analyze: dart analyze --fatal-infos .
   test: melos exec --concurrency=1 --fail-fast --dir-exists=test -- "flutter test"
+  generate_neon: melos exec --scope="neon*" -- dart run build_runner build --delete-conflicting-outputs
```

Comment:
I noticed this also runs on packages that don't use build_runner. How about adding an empty `build.yaml` everywhere where we use build_runner and then filter for that file?

Reply:
Yeah that's bad.

I just cherry picked this from my go_router branch and there every app will be expected to use build_runner for the router. In that case it wouldn't be needed.

Should I change this anyways?

Reply:
Yeah, at least for this PR please. You can remove it later on. But it might happen that some app won't use go_router, right? Like a really simple single page app (i don't know any, but it could happen).